WcaOnRails/app/controllers/payment_controller.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+class PaymentController < ApplicationController
+  def payment_config
+    payment_id = params.require(:payment_id)
+    competition_id = params.require(:competition_id)
+
+    competition = Competition.find(competition_id)
+    stripe_transaction = StripeTransaction.find(payment_id)
+    secret = stripe_transaction.stripe_payment_intent.client_secret
+    render json: { stripe_publishable_key: AppSecrets.STRIPE_PUBLISHABLE_KEY,
+                   connected_account_id: competition.connected_stripe_account_id,
```

Comment:
Also, assuming that this request is trackable in Chrome or Mozilla Dev Tools "Network" tab: Is it safe to pass the `connected_account_id` around? Is that information visible in our current monolith implementation?

Reply:
The stripe microservice needs this to render the stripe UI